### PR TITLE
Remove component adoption labels before initial CSV copy.

### PR DESCRIPTION
**Pull request by benluddy**
Operator component labels were not being stripped before initially
creating a CSV copy (they were only being stripped on subsequent
updates to copies).

This explains why the end-to-end test for copied CSV adoption has been failing sporadically.

### CODE REVIEW

**Comment by benluddy** [comment]
https://github.com/operator-framework/operator-lifecycle-manager/issues/2091

**Comment by benluddy** [comment]
/hold

Looking into a test failure that looks related:

```[Fail] ClusterServiceVersion when a copied csv exists [It] is synchronized with the original csv```

File: pkg/controller/operators/olm/operatorgroup.go [hunk 1]
```diff
@@ -705,79 +705,61 @@ func (a *Operator) copyToNamespace(csv *v1alpha1.ClusterServiceVersion, namespac
...
-		if _, err := a.client.OperatorsV1alpha1().ClusterServiceVersions(namespace).UpdateStatus(context.TODO(), createdCSV, metav1.UpdateOptions{}); err != nil {
-			a.logger.Errorf("Status update for CSV failed: %v", err)
-			return nil, err
-		}
-
-		return createdCSV, nil
-
 	} else if err != nil {
-		logger.WithError(err).Error("couldn't get CSV")
+		a.logger.Errorf("Create for new CSV failed: %v", err)
```
Comment (dinhxuanvu) [inline pkg/controller/operators/olm/operatorgroup.go:748]: No need for `a.` here.

**Review by dinhxuanvu — APPROVED** [review]
One nit. Otherwise, lgtm.
/lgtm

**Comment by benluddy** [comment]
/hold cancel

Should be resolved now -- the cache-get is load-bearing and has to stay.

**Comment by benluddy** [comment]
https://github.com/operator-framework/operator-lifecycle-manager/issues/2091

/retest

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

**Pull request merged**